bin/prettier.js

```diff
@@ -153,20 +153,21 @@ function format(input) {
     const pp = prettier.format(input, options);
     const pppp = prettier.format(pp, options);
     if (pp !== pppp) {
-      process.stdout.write("\n");
-      console.error('prettier(input) !== prettier(prettier(input))');
-      console.error(diff(pp, pppp));
-      process.exitCode = 2;
+      throw "prettier(input) !== prettier(prettier(input))\n" + diff(pp, pppp);
```

Comment:
* Throwing here so we log the filename in the error (`handleError`).
* Throwing a string so we don't get the stack trace.